docs/source/technical_tutorials/dps_tutorial/DPS_runner_template.ipynb

```diff
@@ -0,0 +1,353 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.                version=MAAP_VERSION,
</pre><p>I believe you want <code>ALGO_VERSION</code>, not <code>MAAP_VERSION</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>